extensions/stackdriver/edges/mesh_edges_service_client.cc

```diff
@@ -50,13 +50,23 @@ MeshEdgesServiceClientImpl::MeshEdgesServiceClientImpl(
     RootContext* root_context,
     const ::Extensions::Stackdriver::Common::StackdriverStubOption& stub_option)
     : context_(root_context) {
-  success_callback_ = [](size_t) {
+  Metric export_call(MetricType::Counter, "export_call",
```

Comment:
should the metric be defined elsewhere, as it is used in the log exporter with the same definition?

Reply:
Make sense. Done.